src/server.ts

```diff
@@ -158,7 +158,7 @@ export class RemixI18Next {
     await instance.changeLanguage(locale);
     await instance.loadNamespaces(namespaces);
 
-    return instance.getFixedT(locale, namespaces);
+    return instance.getFixedT(locale, namespaces as string | string[]);
```

Comment:
Here TypeScript was complaining about the `readonly string[]` so I typecast it this way.